### DISCUSSION TITLE
Relay URL issue

### DISCUSSION THREAD

**Discussion by stanlys96** [WalletConnect/walletconnect-monorepo#3480] [Developer Support] [answered]
Hello there. I have an issue. I've implemented to use wallet connect v2, but the sites I've been trying to connect to still don't use relay-url. Thetan Arena, Gunstar, Castle Crush, they still use bridge url, so I can't connect to their marketplace. Anybody has a solution to this? This is the error from my React Native console. Thank you!

<img width="1154" alt="Screen Shot 2023-08-17 at 14 55 55" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/54090219/e542df74-2cbd-4bd9-be6a-da4aabc5cac3">

**Comment by glitch-txs** [comment, marked as answer]
They need to upgrade to v2 on the Dapp side. You can check the errors on their console
